.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xbmc/addons/kodi-addon-dev-kit"]
+	path = xbmc/addons/kodi-addon-dev-kit
+	url = git://github.com/AlwinEsch/kodi-addon-dev-kit.git
```

Comment:
Uhm cmake is trivial, externalproject drives the entire addon build system. The others are manual work. Just google cmake externalproject.